src/app/shared/entry-wizard/entry-wizard.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-progress-bar *ngIf="isLoading$ | async" mode="indeterminate"></mat-progress-bar>
+<div>
+  <p>Toggle repositories on to create workflows from the associated Git repository. Once added, these workflows can be customized.</p>
+  <p>This approach is best for Git repositories with only one workflow.</p>
+  <div>
+    <mat-form-field class="w-100">
+      <mat-label>Git Registry</mat-label>
+      <mat-select [(ngModel)]="selectedGitRegistry" name="registry" (ngModelChange)="getOrganizations($event)">
+        <mat-option *ngFor="let registry of gitRegistries$ | async" [value]="registry">
+          {{ registry }}
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+  </div>
+  <div *ngIf="selectedGitRegistry !== null && (gitOrganizations$ | async)">
+    <mat-form-field class="w-100">
+      <mat-label>Organization</mat-label>
+      <mat-select [(ngModel)]="selectedGitOrganization" name="organization" (ngModelChange)="getRepositories($event)">
+        <mat-option *ngFor="let organization of gitOrganizations$ | async" [value]="organization">
+          {{ organization }}
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+
+    <div fxLayout="column" fxLayoutAlign="start start" fxLayoutGap="20px">
+      <div *ngFor="let repo of gitRepositories$ | async">
```

Comment:
I put this in my screenshot comment already, but I think there should be a mat-label here